.github/workflows/static-root-checks.yml

```
@@ -39,9 +39,12 @@ jobs:
       - name: Check types
         working-directory: packages/react-native-reanimated
         run: yarn type:check:all
-      - name: Lint
+      - name: Lint reanimated
```

I'm submitting the same suggestion again. It looks like it was resolved but for some reason it has been overwritten. Maybe you've force pushed the commits from your local branch?
```suggestion
      - name: Lint react-native-reanimated
```